File: Function/LeapYear.c
#include <stdio.h>
int leapYear (int);
int main()
{
	int year;
	printf("Enter a year to check leap year or not:");
	scanf("%d",&year);
	if (leapYear(year))
	{
		printf("Leap Year");
	}
	else
	{
		printf("Not Leap Year");
	}
}
int leapYear(int y)
{
	return (y%400==0)||((y%4==0)&&(y%100!=0));
			
}

File: Java new/LeapYear.java
import java.util.Scanner;
class LeapYear {
    public static void main(String[] args) {
        Scanner year = new Scanner (System.in);
        System.out.println("Enter a year:");
        int a= year.nextInt();

        if((a%400==0) || (a%100!=0) && (a%4==0))
	    {
		    System.out.println("Leap year");
	    }
	    else
	    {
		    System.out.println("Not leap year");
	    }
    }

}
